terraform/aws/voiceOpsAWS.tf

```diff
@@ -115,12 +158,79 @@ resource "aws_route_table_association" "voiceOpsRta1c" {
 # Then ASGs with sizes
 
 # AWS Launch Configuration Setup
+# K8 Node launch config
+resource "aws_launch_configuration" "voiceOpsNode" {
+    name = "voiceOpsNode"
+    image_id = "ami-8e5743e8"
+    instance_type = "t2.large"
+    key_name= "demo"
+    security_groups = ["${aws_security_group.voiceOpsHTTP.id}"]
+}
 
-## TBD
+# K8 Master launch config
+resource "aws_launch_configuration" "voiceOpsMaster" {
+    name = "voiceOpsMaster"
+    image_id = "ami-8e5743e8"
+    instance_type = "t2.medium"
+    key_name= "demo"
+    security_groups = ["${aws_security_group.voiceOpsHTTP.id}"]
+}
+
+# Bastion host launch config
+resource "aws_launch_configuration" "voiceOpsBastion" {
+    name = "voiceOpsBastion"
+    image_id = "ami-8e5743e8"
+    instance_type = "t2.micro"
+    key_name= "demo"
+    security_groups = ["${aws_security_group.voiceOpsSSH.id}"]
+}
 
 # AWS Auto Scaling Group Setup
 
-## TBD
+# K8 Node ASG
+resource "aws_autoscaling_group" "voiceOpsNodeASG" {
+  availability_zones = ["eu-west-1a", "eu-west-1b", "eu-west-1c"]
```

Comment:
happy to go with 3 ASG's one for each AZ - i will update as per above later today